roq-deployment/development/GCM.log

```diff
@@ -0,0 +1,9 @@
+Starting the  global configuration manager
```

Comment:
This file should not have been committed. Please remove it and add it to the _.gitignore_ file if necessary.